charts/kube-state-metrics/LICENSE

```diff
@@ -0,0 +1,202 @@
+
```

Comment:
Probably remove it since it is the same as the one at root!

Reply:
Will do!

Reply:
In a later PR ?

Reply:
I removed it in this one. :) Thanks for the reminder!